test/tftp/tftp_api_test.rb

```diff
@@ -17,6 +17,17 @@ def setup
     @args = { :pxeconfig => "foo" }
   end
 
+  def test_instantiate_syslinux
+    obj = app.helpers.instantiate "syslinux", "AA:BB:CC:DD:EE:FF"
```

Comment:
It's not much harder to test all available variants, could you modify the test to do that?